bin/run_data_task.py

```diff
@@ -47,7 +47,7 @@
 def main():
     args = parser.parse_args()
 
-    with bootstrap(args.config_file) as env:
+    with bootstrap(args.config_file) as env:  # noqa: PLR1702
```

Comment:
As last time we talked about this is unfornuate that we have to refer to this by code

That's tracked here: https://github.com/astral-sh/ruff/issues/1773


This particular code is for: https://docs.astral.sh/ruff/rules/too-many-nested-blocks/